docs/en/ingest-management/security/enrollment-tokens.asciidoc

```diff
@@ -0,0 +1,89 @@
+[[fleet-enrollment-tokens]]
+= {fleet} enrollment tokens
+
+An enrollment token is an {es} API key that you use to enroll one or more
+{agent}s in {fleet}. The enrollment token enrolls the {agent} in a specific
+agent policy that defines the data to be collected by the agent. You can
+use the token as many times as required. It will remain valid until you revoke
+it.
+
+The enrollment token is used for the initial communication between {agent} and
+{fleet-server}. After the initial connection request from the {agent},
+the {fleet-server} passes two API keys to the {agent}:
+
+* An output API key
++
+This API key is used to send data to {es}. It has the minimal permissions needed
+to ingest all the data specified by the agent policy. If the API key is invalid,
+the {agent} stops ingesting data into {es}.
+
+* A communication API key
++
+This API key is used to communicate with the {fleet-server}. It has only the
+permissions needed to communicate with the {fleet-server}. If the API key is
+invalid, {fleet-server} stops communicating with the {agent}.
+
+// TODO: Convert the following steps to 7.14 after this topic is merged and
+// backported to 7.13.
+
+//QUESTION: Do we want to document the API calls for creating enrollment tokens
```

Comment:
I would expect in most cases there user never has to manually create enrollment tokens but creates policies and with it the enrollment token. I would rather focus on getting the Kibana API docs in place.